Content.Server/Storage/EntitySystems/PickRandomSystem.cs

```diff
@@ -0,0 +1,60 @@
+using Content.Server.Storage.Components;
+using Content.Shared.Database;
+using Content.Shared.Hands.EntitySystems;
+using Content.Shared.Verbs;
+using Robust.Shared.Containers;
+using Robust.Shared.Random;
+using System.Linq;
+
+namespace Content.Server.Storage.EntitySystems;
+
+public sealed class PickRandomSystem : EntitySystem
+{
+    [Dependency] private readonly SharedContainerSystem _container = default!;
+    [Dependency] private readonly SharedHandsSystem _hands = default!;
+    [Dependency] private readonly IRobustRandom _random = default!;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        SubscribeLocalEvent<PickRandomComponent, GetVerbsEvent<AlternativeVerb>>(OnGetAlternativeVerbs);
+    }
+
+    private void OnGetAlternativeVerbs(EntityUid uid, PickRandomComponent comp, GetVerbsEvent<AlternativeVerb> args)
```

Comment:
Verbs should be in shared so there's no delay before it appears in the right click menu.

Reply:
actually uh @0x6273 its not possible since theres no shared StorageComponent so how can this be done

Reply:
then i have to drop the disabled bool since it uses it

Reply:
this is ok to keep on server for now bceause of storage badcode

Reply:
pro ill add a todo that will be addressed in 3 years

Reply:
Couldn't this just be clientside and pick one?

Reply:
isnt client doing rng for important things usually bad

Reply:
The client is opting into this RNG though.